red/api/theme.js

```diff
@@ -128,6 +128,15 @@ module.exports = {
             }
         }
 
+        if (theme.customScripts) {
```

Comment:
The `editorTheme` property already has the ability to inject CSS in the page via `editorTheme.page.css`. This change should be consistent with that and use the property name `editorTheme.page.scripts`.